tests/console/console.go

```diff
@@ -90,11 +90,11 @@ func CheckForTextExpecter(vmi *v1.VirtualMachineInstance, expected []expect.Batc
 	return err
 }
 
-// VmiConsoleRunCommand runs the command line from `command connecting to an already logged in console at vmi
+// RunCommand runs the command line from `command connecting to an already logged in console at vmi
 // and wait `timeout` for command to return.
 // NOTE: The safer version `ExpectBatchWithValidatedSend` is not used here since it does not support cases.
-func VmiConsoleRunCommand(vmi *v1.VirtualMachineInstance, command string, timeout time.Duration) error {
-	err := VmiConsoleExpectBatch(vmi, []expect.Batcher{
+func RunCommand(vmi *v1.VirtualMachineInstance, command string, timeout time.Duration) error {
```

Comment:
I would call it `RunCommandOnVmi`.

Reply:
alternative [suggestion](https://github.com/kubevirt/kubevirt/pull/4296/commits/2df48f83f168f905a87dadef17484ee4caeb77dd#r499649691) made by @qinqon in the first commit: changing the module name `console`-->`vmiconsole` would make it clear enough, without needing the `OnVmi` suffix.